lib/ClangImporter/ImportDecl.cpp

```diff
@@ -5583,8 +5583,7 @@ namespace {
 }
 
 static bool conformsToProtocolInOriginalModule(NominalTypeDecl *nominal,
```

Comment:
Nit: Now that we're not passing in Foundation, this should probably be called something like `conformsToProtocolInFoundationModule()`.